### DISCUSSION TITLE
how to invite the same user to multiple Budibase apps?

### DISCUSSION THREAD

**Discussion by syhu** [Budibase/budibase#14148] [Help]
I'm involved in several startup projects, and have found Budibase to be a great prototyping / productivity tool and really love it!

As I'd like to invite other users to try out our apps, I've discovered that it seems like I "cannot" invite users who already have existing Budibase account and may have created their own workspaces. I would get an "Email already in use. Please use a different email." error.

It seems that each user can ONLY belong to ONE workspace and have access to the apps within that workspace. Is that the case? or are there ways to get around it?

Also, when I tried to invite a user to try out the app by email, that user then has to create a new Budibase account with a workspace, making the invited user unable to join my app and become a user. I've previously successfuly invited others to try our app but now the process doesn't seem to repeat. Am I missing something? 

Will appreciate any feedback / pointer on how to allow user invitations to the apps we created. Thanks!

**Comment by ConorWebb96** [comment]
Hey @syhu,

Users can only exist in 1 instance of Budibase at a time. If you have invited a user to a previous tenant then they are only able to use that email to login to that tenant. If you are setting up multiple accounts for Budibase, then the users will need multiple emails in order to gain access to the new instances.

You can create multiple apps per account. If you already have the users in a tenant you can give them access to other apps as and when you create them.

I hope this makes sense.
